docs/config/README.md

```diff
@@ -2168,7 +2168,7 @@ By default the module will be shown if any of the following conditions are met:
 | ------------------- | ------------------------------------ | ----------------------------------------------------------------------------------------------------- |
 | `format`            | `"via [$symbol($version )]($style)"` | The format for the module.                                                                            |
 | `version_format`    | `"v${raw}"`                          | The version format. Available vars are `raw`, `major`, `minor`, & `patch`                             |
-| `symbol`            | `" "`                               | A format string representing the symbol of Node.js.                                                   |
+| `symbol`            | `" "`                               | A format string representing the symbol of Node.js.                                                   |
```

Comment:
Oh. Oh boy.

![image](https://user-images.githubusercontent.com/4605384/151685262-8c09166a-652c-4c26-ac1b-ff9331667ac0.png)

It seems that the new symbol is part of the NerdFonts CJK overlap issue. See https://github.com/ryanoasis/nerd-fonts/issues/365 for details. In short, several NerdFont symbols bled out of the PUA into the BMP and can override/be overridden by symbols from other human languages.

This is strictly speaking a NerdFonts issue, but I think we should be prepared for some issues about this symbol not appearing correctly on systems with CJK support.

Reply:
I wonder if there are other symbols like this new symbol in the default configuration. Maybe we can list them in the docs.

Reply:
Quite a few, as it turns out. From this system, in the NerdFont preset, it looks like AWS, golang, julia, perl, ruby, rust, and scala are affected. There may be more in other parts of the default config.

Reply:
Ok～

I have a confusion, the new nodejs symbol is e718, it belong the PUA area (U+E000..U+F8FF).

Reply:
it should be displayed correctly.

Reply:
Oh.

Hm, I wonder if this system is actually misconfigured then? I'm using Pop_OS's default fontconfigs, which I assumed should be correct, but maybe those are broken...

Reply:
It doesn't render for me on Windows in the browser, and I have some CJK fonts installed.

Reply:
Which browser are you using? It seems to me that it should render correctly because the browser loads the nerd-font.woff2 font.

Reply:
> It doesn't render for me on Windows in the browser, and I have some CJK fonts installed.

Could you check the nerd-font.woff2 whether has been loaded?
![image](https://user-images.githubusercontent.com/33021497/153182527-6ec85b3a-491f-4659-bbce-dc710468f1e7.png)
I ran into some unexpected issues when the nerd-font.woff2 has not been loaded.
![image](https://user-images.githubusercontent.com/33021497/153184545-2de003b8-940c-4f45-b35c-b4abdf2aad88.png)
![image](https://user-images.githubusercontent.com/33021497/153184690-cf215368-dab3-42b5-9080-b7f7f20de33d.png)